src/Sarif.Viewer.VisualStudio/Themes/CallTrees.xaml

```diff
@@ -22,10 +22,10 @@
                         <TreeView.ItemTemplate>
                             <HierarchicalDataTemplate DataType="x:Type local:CallTreeNode" ItemsSource="{Binding Children}">
                                 <StackPanel Orientation="Horizontal">
-                                    <TextBlock Text="!  " FontWeight="Bold" Foreground="Red" 
+                                    <TextBlock Text="! " FontWeight="Bold" Foreground="Red" Background="Yellow" FontFamily="Consolas"
                                                Visibility="{Binding Location.Importance, 
```

Comment:
Use the importanceToBackgroundConverter, rather than hard-coding "Yellow". If we changed the color we use in the converter, we would have to remember to change it here, too.